templates/example_flowsom_clustering.ipynb

```diff
@@ -47,9 +47,13 @@
    "outputs": [],
```

Comment:
<p>I think we can remove channels as an argument to create_pixel_matrix, and preprocess all of the channels. We can then move the cell directly above this cell defining the subset_channels to below this cell, since it will only be needed by the train_som function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>